### ISSUE TITLE
Sending IRC commands

### ISSUE THREAD

**Issue by rahatarmanahmed** [aeirola/slack-irc-client#18]
Would be nice if there was a way to send an IRC command from slack. I know that the point of this is so you don't have to deal with the garbage that is IRC but sometimes you just need to be able to use a command.

**Comment by aeirola** [comment]
True, would be nice. I guess this could be under the `slackbot` channel which already acts like some kind of status window. Unfortunately I've been having problems finding time for this project, hope to bring it back on track soon.

**Comment by autostatic** [comment]
+1 for the possibility to send IRC commands.
